server/routes/induction/common/previousWorkExperienceDetailController.ts

```diff
@@ -89,11 +89,11 @@ const toPreviousWorkExperienceDetailForm = (
   inductionDto: InductionDto,
   typeOfWorkExperience: TypeOfWorkExperienceValue,
 ): PreviousWorkExperienceDetailForm => {
-  const previousWorkExperience = inductionDto.previousWorkExperiences.experiences.find(
+  const previousWorkExperience = inductionDto.previousWorkExperiences?.experiences.find(
```

Comment:
Belt'n'braces - just makes sure we dont NPE or equivalent when trying to render the page where there is no job role/details for the given work experience type